braille-printer-app/LICENCE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
You can remove LICENSE and NOTICE from this subdir. they are already in the main directory.